salsah/src/test/scala/org.knora.salsah/browser/SearchAndEditSpec.scala

```diff
@@ -93,25 +85,25 @@ class SearchAndEditSpec extends SalsahSpec {
         }
 
         "have the correct title" in {
-            page.load()
+            page.open()
             page.getPageTitle should be ("System for Annotation and Linkage of Sources in Arts and Humanities")
 
         }
 
         "log in as an Incunabula project user" in {
 
-            page.load()
-            page.doLogin(email = incunabulaUserEmail, password = testPassword, fullName = incunabulaUserFullName)
+            page.open()
+            page.doLogin(email = incunabulaUserEmail, password = testPassword, fullName = incunabulaUserFullName, page)
```

Comment:
Why do you pass `page` to `page.doLogin`? Wouldn't that just be `this` inside the method?